src/h2load.h

```diff
@@ -215,6 +217,14 @@ struct Stats {
 
 enum ClientState { CLIENT_IDLE, CLIENT_CONNECTED };
 
+// This variable tells us whether the client is in warmup phase or not or is over
+// 0 - Not in warm-up phase, this is the initial state in timing experiment
+// 1 - In warmup phase. This happens after the first connect. 
+//   - All statistics are skipped/reversed in this phase
+// 2 - Main duration phase, in timing experiment; Otherwise, the normal phase
+// 3 - Main duration is over
+enum Phase { INITIAL_IDLE, WARM_UP, MAIN_DURATION, DURATION_OVER };
```

Comment:
Didn't know about this. Makes sense.
https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum